src/api.ts

```diff
@@ -47,15 +48,18 @@ router.get('/', (req, res) => {
 });
 
 router.get('/scores/:proposalId', async (req, res) => {
-  const { proposalId } = req.params;
-  try {
-    const result = await serve(proposalId, updateProposalAndVotes, [proposalId]);
-    return res.json({ result });
-  } catch (e) {
-    capture(e);
-    log.warn(`[api] updateProposalAndVotes() failed ${proposalId}, ${JSON.stringify(e)}`);
-    return res.json({ error: 'failed', message: e });
-  }
+  // Delay the score finalization to allow some time for last minute votes to complete
+  setTimeout(async () => {
+    const { proposalId } = req.params;
+    try {
+      await serve(proposalId, updateProposalAndVotes, [proposalId]);
+    } catch (e) {
+      capture(e);
+      log.warn(`[api] updateProposalAndVotes() failed ${proposalId}, ${JSON.stringify(e)}`);
+    }
+  }, SCORE_FINALIZATION_DELAY);
+
+  return res.json({ result: 'pending' });
```

Comment:
I think we should do it inside `updateProposalAndVotes` based on the proposal end time, here we are adding wait time on all requests.

Reply:
better to use `snapshot.utils.sleep`?

Reply:
Moved the delay to inside `updateProposalAndVotes`

Now, any calls to this function will have a delay, and will process at the earliest, 1 min after the proposal end.

The main API call has been moved back to an sync call, and the call from webhook may taken up to 1min to respond. Upside is that the response is back to a boolean, and will not break UI calls